api/tests/routes/backoffice/finance_test.py

```diff
@@ -688,6 +691,37 @@ def test_not_validate_incident(self, authenticated_client, initial_status):
         assert finance_models.FinanceEvent.query.count() == 0
         assert len(mails_testing.outbox) == 0
 
+    @pytest.mark.parametrize(
+        "exception_type,expected_error_msg",
+        [
+            (crg_exceptions.ExternalBookingException, "Une erreur s'est produite : Test"),
```

Comment:
Détail : plus logique :
```suggestion
            (cgr_exceptions.CGRAPIException, "Une erreur s'est produite : Test"),
```